Dockerfile

```diff
@@ -1,20 +1,32 @@
 FROM ubuntu:14.04
-MAINTAINER Loïc PORTE
-RUN apt-get update && apt-get install git python python-dev\
+MAINTAINER PMSIpilot
```

Comment:
Original author must be kept. I would have removed PMSIpilot in favor of your name!